webapps/frontend/app/viewmodels/services/storagerouter.js

```diff
@@ -47,7 +46,34 @@ define([
             return api.post('storagerouters/' + storageRouterGuid + '/get_metadata')
                 .then(shared.tasks.wait)
         };
-
+        /**
+         * Fetches support info of a StorageRouter
+         * @param storageRouterGuid: Guid of the StorageRouter
+         * @return {Promise<T>}
+         */
+        self.getSupportSettingsHandle = function(storageRouterGuid) {
```

Comment:
Handles are used to limit api calls. This should be getSupportInfo instead.
The handles is the XHRRequest that the api.get returns. It is then used to check if the xhr was already resolved or not before re-querying.